compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/OwnedLayerManager.kt

```diff
@@ -40,4 +40,8 @@ internal interface OwnedLayerManager {
     fun notifyLayerIsDirty(layer: OwnedLayer, isDirty: Boolean)
 
     fun invalidate()
+
+    val isArrEnabled: Boolean
```

Comment:
It's a global flag, why it is required to be in this interface?

Reply:
The implementation of this interface is not platform-specific. So I'm still not sure if it has to be here.

The usages are like "fire and forget" everywhere, so platforms might just ignore calls with additional contitions in `PlatformContext` implementation

Reply:
I have removed the value from the interface as it is no longer needed there.